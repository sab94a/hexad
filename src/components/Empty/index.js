// @flow 

import React from 'react';

const Empty = () => (
    <div>There are no any items..</div>
);

export default Empty;
