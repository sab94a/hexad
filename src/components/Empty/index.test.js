import React from 'react';
import { shallow } from 'enzyme';
import { shallowToJson } from 'enzyme-to-json';
import Empty from './'

describe('Loader', () => {
    it('Should render correctly', () => {
        const component = shallow(<Empty />);
        expect(shallowToJson(component)).toMatchSnapshot();
    });
})
