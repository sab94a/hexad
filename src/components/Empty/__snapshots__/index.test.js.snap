// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Loader Should render correctly 1`] = `
<div>
  There are no any items..
</div>
`;
